syntax = "proto3";
package io.dmtri.scooters;

message Vector {
    int32 x = 1;
    int32 y = 2;
}

message ScooterTelemetry {
    string scooter_id = 1;
    uint64 timestamp = 2;
    Vector coordinates = 3;
    Vector speed = 4;
}

message ScooterTelemetryResponse {
    string message = 1;
}